### DISCUSSION TITLE
[1.6.3] 2022-06-28

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#317] [General]
# Wireless
* [HID] Fixup axes min, neutral & max values

# Wired
* [N64] Reduce scope of N64 digital workaround to N64 port 1 only

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.6.3'>[1.6.3] 2022-06-28</a>.</em>

**Comment by Luizcsf** [comment]
First of all, thank you for your excellent work!

I want to install blue retro internally on gamecube, but i only see the gc_external.bin.
Can i use this bin file for internal installation too?

**Reply by darthcloud** [reply]
Internal install is not officially supported yet for GC.

Some do it anyway, but be advised wired controller will likely not work anymore. (if it does you are lucky)

**Reply by Luizcsf** [reply]
Ok, thank you!
Waiting for new releases!

**Comment by EduardoYagami** [comment]
Hello friend, how are you?
I wonder if there is no more update or what file should currently be used to install internally on the Nintendo Wii Game Cube ports. I currently use 1.6.1 bootloader and partition table, (Current version at the time I installed it) but I still use "BlueRetro_gc_spiffs.bin" from version 1.2.1. Since this file no longer exists in recent versions and aparentely only this file works on Wii's Game cube ports. Thanks!

**Reply by darthcloud** [reply]
It's likely BlueRetro_gc_external.bin, the only one with "gc" (**G**ame**C**ube) in the name

**Reply by EduardoYagami** [reply]
Hi, thanks for replying. In fact, this "BlueRetro_gc_external.bin" file doesn't work on Wii at least since version 1.6.1, which I tested. The 1.2.1 works. It made me think it would no longer be this current "GC" file. If it doesn't work, what could it be?

**Reply by darthcloud** [reply]
use 1.6.3

**Reply by EduardoYagami** [reply]
Hi there!  This 1.6.3 worked fine. Sorry, I was only able to test this rev a few days ago. thank you so much!
